example1/shapes/areaCalculator.py

```
@@ -0,0 +1,35 @@
+import math
+
+class AreaCalculator:
+
+    def __init__(self, shapes=[]) -> None:
```

Ruff (0.8.0)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)